sympy/printing/julia.py

```diff
@@ -405,7 +404,15 @@ def _print_HadamardPower(self, expr):
             self.parenthesize(expr.exp, PREC)
             ])
 
-    # Note: as of 2015, Julia doesn't have spherical Bessel functions
+    def _print_Rational(self, expr):
+        if expr.q == 1:
+            return str(expr.p)
+        else:
+            if self._settings.get("sympy_integers", False):
+                return "S(%s) / %s" % (expr.p, expr.q)
```

Comment:
I am not sure where this comes from? `S` is pretty SymPy-specific and I doubt that Julia has that, but I can be wrong of course...

Reply:
Thanks for reminding, I copied it from `StrPrinter`, this should be deleted in `JuliaCodePrinter`.